src/LPOracle.sol

```diff
@@ -128,7 +128,7 @@ contract LPOracle is AggregatorV3Interface {
     |*  # INTERNAL HELPERS                                      *|
     |*----------------------------------------------------------*/
 
-    /// @notice Retrieves latest price data from Chainlink feeds and adjusts for decimals.
+    /// @notice Retrieves latest price data from Chainlink feeds and normalise answers to 18 decimals.
```

Comment:
Doesn't the `decimals()` function also need to be updated? I think it might be a nice addition to the unit test to also assert what `decimals()` returns next to when we assertApproxEq on the result.